### DISCUSSION TITLE
Production readiness

### DISCUSSION THREAD

**Discussion by anthonycorbacho** [wind-c/comqtt#14] [Q&A]
Hi, this project looks very interesting and I am considering using it as a replacement for HiveMQ.

My question is around production readiness;

- Is there any company using this in cluster mode with more than 80k active connections?
- How ready is the clustering functionality and scalability?

**Comment by wind-c** [comment]
The number of active connections larger than 80K is not large and can be supported by a single machine. However, for scalability and stability, it is better to use a cluster, a low-configuration cluster with at least 3 machines. You can test by simulating real business scenarios. I do not currently have a large connection scenario and would like users to test and provide scenario cases.

**Comment by anthonycorbacho** [comment]
So basically our use case is that we will have ~80k device that is sending multiple events every 10s so I was hopping to have insight about handling this load.

I will check with my team if we want to give a try, thank you for your kind answer!

**Comment by SimonMacIntyre** [comment]
@anthonycorbacho  Curious where you landed here! We are also evaluating using this project

**Reply by anthonycorbacho** [reply]
Didnt continue because of a layoff so yeah...
